cisco_aci/datadog_checks/cisco_aci/data/conf.yaml.example

```diff
@@ -1,49 +1,97 @@
-# To capture Cisco ACI metrics you need to install the Datadog Agent on one of your hosts.
+init_config:
+
+  ## @param min_collection_interval - integer - optional - default: 180
+  ## This check makes a lot of API calls
+  ## it could sometimes help to add a minimum collection interval
+  #
+  #  min_collection_interval: 180
```

Comment:
I moved this parameter in the init_config.. (and created the init_config section) to be consistent with other checks, is it possible with this integration?

Reply:
This parameter should indeed be in the `instance`